src/transformers/models/bridgetower/image_processing_bridgetower_fast.py

```diff
@@ -165,13 +165,18 @@ def resize(
             raise ValueError(f"The `size` dictionary must contain the key `shortest_edge`. Got {size.keys()}")
         shorter = size.shortest_edge
         longer = int(1333 / 800 * shorter)
-        output_size = get_resize_output_image_size(
+        output_height, output_width = get_resize_output_image_size(
```

Comment:
could you explain why such changes here and below?

Reply:
The idea here is to use the parent's class `resize` function rather than `torchvision` equivalent directly. To do that, we unpack the `output_size` to then use the `output_height` and `output_width` to build a SizeDict which is needed for `super().resize`